tests/model_fields/tests.py

```diff
@@ -211,13 +211,13 @@ def test_abstract_model_pending_lookups(self):
         to_key = ('missing', 'FK')
         fk_lookup = (AbstractForeignFieldsModel, opts.get_field('fk'))
         self.assertFalse(
-            any(lookup[0:2] == fk_lookup for lookup in opts.apps._pending_lookups.get(to_key, [])),
+            any(lookup[0:2] == fk_lookup for lookup in opts.apps._pending_operations.get(to_key, [])),
```

Comment:
In the interest of leaving _pending_operations untouched, could we leave these ``get`` calls as-is?